app/config.py

```diff
@@ -0,0 +1,9 @@
+import os
+from dotenv import load_dotenv
+load_dotenv('../.env')
+
+POSTGRES_USER = os.getenv("POSTGRES_USER")
+POSTGRES_PASSWORD = os.getenv("POSTGRES_PASSWORD")
+POSTGRES_HOST = os.getenv("POSTGRES_HOST")
+POSTGRES_PORT = os.getenv("POSTGRES_PORT")
+POSTGRES_DB = os.getenv("POSTGRES_DB")
```

Comment:
The idea would be to keep the call of all variables just inside that *config* file with like this:
``` 
DB_URI = (
     f"postgresql://{DB_USER}:{DB_PASSWORD}"
     f"@{DB_HOST}:{DB_PORT}/{DB_NAME}"
)
``` 
and in the modules use like this:

engine = create_engine(config.DB_URI)

Reply:
ok